pkg/services/notifications/webhook.go

```diff
@@ -26,6 +26,10 @@ type Webhook struct {
 }
 
 var netTransport = &http.Transport{
+	TLSClientConfig: &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
this seems like a reduction in security that not everyone would be ok with, is it required for this fix?

Reply:
Yes, I based my approach on the work of BrandonArp here https://github.com/grafana/grafana/commit/cffbcb504d884022f41fc48a6099d15018e01863
The request is from grafana to a distant server not the reverse

Reply:
Hm. I thought we removed all those some time ago. `InsureSkipVerify` should not be true by default.

Reply:
Ok, I thought the the both was related - InsecureSkipVerify: true, and Renegotiation -, I can push a merge without this entry ?

Reply:
Yes we did the commit you refer to is very old. 
https://github.com/grafana/grafana/blob/master/pkg/models/datasource_cache.go#L58

Reply:
please do :)

Reply:
Yes I see, unfortunately tlsSkipVerify is a value from datasource, https://github.com/grafana/grafana/pull/9377 not global
So I guess the best think to do make a checkbox like as datasource (but beyond my knowledge of go)

InsureSkipVerify is not related with Renegotiation: tls.RenegotiateFreelyAsClient,

So I can push a request with only this entry ? Anyway tls.RenegotiateFreelyAsClient is a default value in your code now

Reply:
only pushing `Renegotiation: tls.RenegotiateFreelyAsClient,` makes total sense.

Reply:
And add a checkbox to your todo list for notification tab :)

Reply:
Ok pushed

Reply:
you need to import "crypto/tls" in the top of the file

Reply:
ok done

Reply:
you need to run `gofmt -s -w ./pkg` in the root of the grafana folder to get the correct formating.

Reply:
Ok, there are also 3 others files "fixed"